modules/eco-poison-pill-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="troubleshooting-poison-pill-operator_{context}"]
+= Troubleshooting the Poison Pill Operator
+
+
+[id="general-troubleshooting-poison-pill-operator_{context}"]
+== General Troubleshooting 
+
+Issue::
+You want to troubleshoot issues with the Poison Pill Operator.
+
+Resolution::
+Check the Operator logs.
+
+[id="checking-daemon-set_{context}"]
+== Checking the daemon set 
+Issue:: The Poison Pill Operator is installed but the daemon set is not available.
+
+Resolution:: Check the Operator logs for errors or warnings.
+
+[id="unsuccessful_remediation{context}"]
+== Unsuccessful remediation 
+Issue:: An unhealthy node was not remediated.
+
+Resolution:: Check and verify if the `PoisonPillRemediation` CR is created. Run the following command:
++
+[source,terminal]
+----
+$ oc get namespace <machinehealthcheck>
+----
++
+If the `MachineHealthCheck` controller did not create the `PoisonPillRemediation` CR when the node turned unhealthy, check the logs of the `MachineHealthCheck` controller.
```

Comment:
maybe it is worth mentioning that a PoisonPillRemediationTemplate resource needs to exists (to be created manually)
in the cluster, for the machine health check to create the ppr